.env

```diff
@@ -1,4 +1,5 @@
-APP_ENV=prod
+APP_ENV=dev
+APP_VERSION=0.0.1-SNAPSHOT
 HUB=zufarexplainedit
```

Comment:
if you use prefix "APP", it make sense to use it in all other env variables:
- APP_DOCKER_HUB
- APP_DOCKER_TAG
- APP_DATASOURCE_URL

Reply:
The prefix "APP" denotes that the variable is directly related to the application itself. Other variables like "HUB" or "TAG" could be more generic and pertain to things outside the app, like CI/CD or Docker.
Also, not using a prefix for certain variables keeps things flexible for when we might have variables that don't neatly fit under the "APP_" prefix.